test/core/security/roleRepository.test.js

```diff
@@ -695,7 +722,8 @@ describe('Test: security/roleRepository', () => {
         }
       };
 
-      should(() => roleRepository.checkRolePluginsRights(role)).not.throw();
+      should(async () => await roleRepository.checkRolePluginsRights(role)).not.throw();
```

Comment:
possible false positive, you have to "await" the should method since you pass it an async function

but same as before, this is simpler: 

```suggestion
      await roleRepository.checkRolePluginsRights(role);
```